frontend/src/components/landing/Landing.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Row, Col } from 'react-bootstrap'
+import Button from 'react-bootstrap/Button'
+import './Landing.css'
+import NavBar from './LandingNav'
+
+export default function LandingPage(){
+
+  return (
+    <div style={{height:"100vh", overflow:"hidden"}}>
+      <NavBar/>
+    <div className='container pt-5'>
+      <Row className='mb-3 mt-5'>
+        <Col className="text-center">
+        <div>
+          <h1 className='' style={{ fontWeight: 'bold' }}>Click Your Way <br/> To Success</h1>
```

Comment:
Looks good, but we can make a different font to make it prettier in the future sprint.